docs/reference/ml/df-analytics/apis/get-inference-trained-model.asciidoc

```diff
@@ -133,8 +136,86 @@ The license level of the trained model.
 
 `metadata`:::
 (object)
-An object containing metadata about the trained model. For example, models 
+An object containing metadata about the trained model. For example, models
 created by {dfanalytics} contain `analysis_config` and `input` objects.
+.Properties of metadata
+[%collapsible%open]
+=====
+`total_feature_importance`:::
+(array)
+An array of the total feature importance for each training feature used from
```

Comment:
> An array of the total feature importance for each training feature used from

AFAIK we've never defined a "training feature".  Can we just say feature? If not, this term needs to be explained.